sparse_strips/vello_cpu/src/fine.rs

```diff
@@ -1,12 +1,12 @@
 // Copyright 2025 the Vello Authors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
+//! Fine rasterization.
+//!
```

Comment:
```suggestion
```
or:
```suggestion
//!
//! Fine rasterization converts from tiles containing lines into strips with alpha values for each pixel.
```

(I don't know if that description is true, which is why we should have that description :) but just to give a flavour of what I'd like to see)